localization/ndt_scan_matcher/README.md

```diff
@@ -255,3 +255,79 @@ initial_pose_offset_model is rotated around (x,y) = (0,0) in the direction of th
+| --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- | ------------------------------- | ------------------------------------------------------------------------------------------------------- |
+| `timer_callback_time_stamp`                         | the time stamp of timer_callback calling                                                                                                                                                                                                                | none                                                                                                     | none                            | none                                                                                                    |
+| `is_activated`                                      | whether the node is in the "activate" state or not                                                                                                                                                                                                      | "activate" state                                                                                         | not "activate" state            | none                                                                                                    |
+| `is_set_last_update_position`                       | whether the `last_update_position` is set or not                                                                                                                                                                                                        | set                                                                                                      | not set                         | none                                                                                                    |
+| `distance_last_update_position_to_current_position` | the distance of `last_update_position` to current position                                                                                                                                                                                              | (the distance + `dynamic_map_loading.lidar_radius`) is **smaller** than `dynamic_map_loading.map_radius` | none                            | (the distance + `dynamic_map_loading.lidar_radius`) is **larger** than `dynamic_map_loading.map_radius` |
```

Comment:
Strictly speaking, it is unclear what will happen if it is exactly equal to distance.